src/Castle.Windsor/MicroKernel/ComponentActivator/DefaultComponentActivator.cs

```diff
@@ -173,7 +173,7 @@ protected object CreateInstanceCore(ConstructorCandidate constructor, object[] a
 			return instance;
 		}
 
-#if (!SILVERLIGHT)
+#if FEATURE_SECURITY_PERMISSIONS
```

Comment:
Maybe it would be better to use `FEATURE_SERIALIZATION` or `FEATURE_REMOTING` since the core of why this can't work on .NET Standard is `FormatterServices.GetUninitializedObject` which is for binary serialization.